lib/context.js

```diff
@@ -9,7 +9,9 @@ const {
   kRequest,
   kBodyLimit,
   kLogLevel,
-  kContentTypeParser
+  kContentTypeParser,
+  kRequestValidateFns,
```

Comment:
These symbol should be more expressive

```suggestion
  kRequestValidateWeakFns,
```